static/less/shared-components.less

```diff
@@ -438,6 +438,15 @@ table.table.key-value {
   }
 }
 
+.theme-dark {
+  .loading {
+    .loading-indicator {
+      // surface200
```

Comment:
thank you for leaving the comments, makes finding future references much easier!